e2e/pull/pull.go

```diff
@@ -433,6 +433,47 @@ func orasPushNoCheck(file, ref string) error {
 	return nil
 }
 
+func (c *ctx) testPullDisableCacheCmd(t *testing.T) {
+	cacheDir, err := ioutil.TempDir("", "e2e-imgcache-")
+	if err != nil {
+		t.Fatalf("failed to create temporary directory: %s", err)
+	}
+	defer func() {
+		err := os.RemoveAll(cacheDir)
+		if err != nil {
+			t.Fatalf("failed to delete temporary directory %s: %s", cacheDir, err)
+		}
+	}()
+
+	c.env.TestDir, err = ioutil.TempDir("", "")
+	if err != nil {
+		t.Fatalf("failed to create temporary directory: %s", err)
+	}
+	defer func() {
+		err := os.RemoveAll(c.env.TestDir)
+		if err != nil {
+			t.Fatalf("failed to delete temporary directory %s: %s", c.env.TestDir, err)
+		}
+	}()
+
+	imgName := "testImg.sif"
+	imgPath := filepath.Join(c.env.TestDir, imgName)
+	cmdArgs := []string{"--disable-cache", imgPath, "library://alpine:latest"}
+
+	c.env.RunSingularity(
+		t,
+		e2e.WithPrivileges(false),
+		e2e.WithCommand("pull"),
+		e2e.WithArgs(cmdArgs...),
+		e2e.ExpectExit(0),
+	)
+
+	cacheEntryPath := filepath.Join(cacheDir, "cache")
```

Comment:
How do you tell singularity that you want this to be the cache dir? _maybe I'm I missing something?_

Reply:
Looks like a duplicate of @jscook2345's comment: https://github.com/sylabs/singularity/pull/4212#discussion_r314496306